test/jasmine/tests/geo_test.js

```diff
@@ -2066,4 +2066,31 @@ describe('Test geo zoom/pan/drag interactions:', function() {
         .catch(failTest)
         .then(done);
     });
+
+    describe('plotly_relayouting', function() {
```

Comment:
Geo subplots have three distinct zoom handlers (depending on the projection used). We should add one test per zoom handler. 

Note also, that we don't need to test `dragmode: 'zoom'` and `dragmode: 'pan'`, both dragmode do the same thing as geo subplot do not support zoombox interactions (yet).

Reply:
> We should add one test per zoom handler.

Similar to:

https://github.com/plotly/plotly.js/blob/7a66dafc5e2b38a907b7757849240f432d6e2372/test/jasmine/tests/geo_test.js#L1650-L1651

https://github.com/plotly/plotly.js/blob/7a66dafc5e2b38a907b7757849240f432d6e2372/test/jasmine/tests/geo_test.js#L1750-L1751

https://github.com/plotly/plotly.js/blob/7a66dafc5e2b38a907b7757849240f432d6e2372/test/jasmine/tests/geo_test.js#L1839-L1840

Reply:
Done in 2545746698588ac3f3edaa91bec2a1ac98ac33b3

Reply:
Awesome :tada: